internal/storage/v2/clickhouse/config.go

```diff
@@ -37,3 +51,18 @@ func (cfg *Configuration) Validate() error {
 	_, err := govalidator.ValidateStruct(cfg)
 	return err
 }
+
+func (cfg *Configuration) applyDefaults() {
```

Comment:
@yurishkuro For a lot of the ClickHouse configurations - the driver has its own defaults. Should we let the driver handle those or do we want to define our own?